### PR TITLE
First branch

### PR DESCRIPTION
**Pull request by IrinaPich**

### CODE REVIEW

File: lib/oop/computeShapes.dart [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import 'dart:io';
 import 'package:flutter_work/oop/shapeHierarchy.dart';
 
-void readShapeList() {
+/*void readShapeList() {
```
Comment (gilcu2) [inline lib/oop/computeShapes.dart:4]: Don't left litter in the code

File: lib/oop/shapeHierarchy.dart [hunk 2]
```diff
@@ -1,15 +1,65 @@
...
+  double c;
+  @override double perimeter() {
+    return a + b + c;
+  }
+}
 
+class EquilateralTriangle extends Triangle {
+  EquilateralTriangle(String name, double side):super(name, side, side, side);
+  @override double area() {
+    double h = a * sqrt(3) / 2;
```
Comment (gilcu2) [inline lib/oop/shapeHierarchy.dart]: Improve name of variables

File: lib/oop/shapeHierarchy.dart [hunk 3]
```diff
@@ -1,15 +1,65 @@
...
+    return h * a / 2;
+  }
 
+}
 
-//void calculateFigures(List figuresList) {
+/*class IsoscelesTriangle extends Triangle {
```
Comment (gilcu2) [inline lib/oop/shapeHierarchy.dart:32]: litter in code

File: test/equilateral_triangle_test.dart [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import 'package:test/test.dart';
+import 'package:flutter_work/oop/shapeHierarchy.dart';
+import 'dart:math';
+
+void main() {
+  group('Test Equilateral Triangle Class', () {
+    test('Perimeter for each side = 3', () {
```
Comment (gilcu2) [inline test/equilateral_triangle_test.dart:8]: A test must be structured in Given, When, Then parts. For example:
// Given trianglr
...
// When compute perimeter
...
// Then value must be the expected

File: test/equilateral_triangle_test.dart [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+      double side = 3;
+      var t = EquilateralTriangle('Equilateral', side);
+      var perimeter = t.perimeter();
+      expect(perimeter, equals(9));
```
Comment (gilcu2) [inline test/equilateral_triangle_test.dart]: Don't put values on the fly without meaning. In this case, you must declare a variable expected_perimeter=9

File: test/equilateral_triangle_test.dart [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+    });
+    test('Area for each side = 4', () {
+      double side = 4;
```
Comment (gilcu2) [inline test/equilateral_triangle_test.dart:18]: Same than previous commet tests

**Review by gilcu2 — CHANGES_REQUESTED** [review]

File: lib/oop/computeShapes.dart (hunk 1 above)
Comment (IrinaPich) [reply lib/oop/computeShapes.dart:4]: The code will be used later

File: lib/oop/shapeHierarchy.dart (hunk 3 above)
Comment (IrinaPich) [reply lib/oop/shapeHierarchy.dart:32]: The code will be used later

File: lib/oop/shapeHierarchy.dart (hunk 2 above)
Comment (IrinaPich) [reply lib/oop/shapeHierarchy.dart]: Done

File: test/equilateral_triangle_test.dart (hunk 5 above)
Comment (IrinaPich) [reply test/equilateral_triangle_test.dart]: Done

File: test/equilateral_triangle_test.dart (hunk 4 above)
Comment (IrinaPich) [reply test/equilateral_triangle_test.dart:8]: Done

File: test/equilateral_triangle_test.dart (hunk 6 above)
Comment (IrinaPich) [reply test/equilateral_triangle_test.dart:18]: Done

**Comment by IrinaPich** [comment]
New pull request included all changes

**Pull request closed without merging**